src/test/java/com/github/dockerjava/utils/ContainerUtils.java

```diff
@@ -0,0 +1,70 @@
+package com.github.dockerjava.utils;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.command.CreateContainerResponse;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Container cmd utils
+ */
+public class ContainerUtils {
+
+    private ContainerUtils() {
+    }
+
+    /**
+     * Start specified container
```

Comment:
starts and ensures that it running, but running is also questionable because it may stop in any time.